server/scripts/import-tests/index.js

```diff
@@ -114,12 +109,13 @@ const importTestPlanVersions = async () => {
         // Gets the next ID and increments the ID counter in Postgres
         // Needed to create the testIds - see LocationOfDataId.js for more info
         const testPlanVersionId = (
-            await client.query(
+            await sequelize.query(
                 `SELECT nextval(
                     pg_get_serial_sequence('"TestPlanVersion"', 'id')
-                )`
+                )`,
+                { transaction }
             )
-        ).rows[0].nextval;
+        )[0][0].nextval;
```

Comment:
This might be confusing, can we use some additional defined parameters here?

Reply:
Yeah I was considering doing it that way. Done.

Reply:
Thanks! Appreciate that change being done and easier to follow than what is currently on `main` as well